tracing-opentelemetry/CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+# 0.5.0 (June 2, 2020)
+
```

Comment:
Can we provide some kind of list of the breaking changes or some upgrade notes?

Reply:
The small breaking change is in the `opentelemetry` dependency and should only be for users who are making custom samplers. I changed the section title to breaking changes so folks know they have to upgrade, think it's wroth documenting more here?

Reply:
That's probably fine.